ui/pages/confirmations/components/confirm/scroll-to-bottom/scroll-to-bottom.tsx

```diff
@@ -21,7 +21,7 @@ import { usePrevious } from '../../../../../hooks/usePrevious';
 import { useScrollRequired } from '../../../../../hooks/useScrollRequired';
 import { useConfirmContext } from '../../../context/confirm';
 import { selectConfirmationAdvancedDetailsOpen } from '../../../selectors/preferences';
-import { REDESIGN_DEV_TRANSACTION_TYPES } from '../../../utils';
+import { REDESIGN_DEV_TRANSACTION_TYPES } from '../../../../../../shared/modules/confirmation.utils';
```

Comment:
Should we be avoiding direct checks against the constant and instead rely on the utility methods for consistency and extensibility?